app/models/employee.rb

```diff
@@ -12,6 +14,11 @@ class Employee < ApplicationRecord
   validate :date_of_birth_not_younger_than_15_years
   validates :place_of_birth, presence: true, length: {minimum: 2, maximum: 30}
   validates :home_address, presence: true, length: {minimum: 2, maximum: 30}
+  validate :department_employee_limit
+  
+    def full_name
```

Comment:
This method doesn't really have the right place. The model side should have everything related to CRUD operations. I would move it to the decorator.

Reply:
but don't we validate creation with this method?

Reply:
with `full_name` ?